src/scripts/interface/cellHighlightRenderer.ts

```
@@ -74,13 +98,138 @@ export class CellHighlightRenderer {
+    );
+  }
+
+  private render(selection: RSelectionState) {
+    const context = this._cellContext;
+    const boardSize = InterfaceConstants.BoardSize;
+
+    context.clearRect(0, 0, boardSize, boardSize);
+
+    const hasSelectedPiece = selection.selectedPiece !== null;
+
+    const anyAttackMoves = selection.moves !== null && selection.moves[0].isAttackMove;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)